src/cms/content-menu.md

```diff
@@ -39,10 +39,10 @@ Create rich, [interactive content]({% link cms/dynamic-blocks.md %}) that is dri
 
 ## Media
 
-[Enhanced Media Gallery]({% link cms/enhanced-media-gallery.md %}) helps Merchants to organize and manage their media files in the admin panel.
+[Media Gallery]({% link cms/media-gallery.md %}) helps Merchants to organize and manage their media files in the admin panel.
```

Comment:
most references have "new media gallery".    is there a plan to remove "new" for subsequent releases?  is it necessary now?

Reply:
We will remove "new" later after they remove the "old" functionality. Olga will let us know when she wants to make that transition. On this one, the ""new" isn't part of the formal name and it seemed weird to add lowercase "new".